src/spec/context.cr

```diff
@@ -183,29 +195,23 @@ module Spec
 
   # :nodoc:
   class NestedContext < Context
-    getter parent : Context
-    getter description : String
-    getter file : String
-    getter line : Int32
+    include Item
 
-    def initialize(@description : String, @file, @line, @parent)
-    end
+    getter! parent : Context
```

Comment:
I note that both Context and Example have a parent. Could/Should it be moved into Item?